Content.Shared/_RMC14/Camera/RMCStaticZoomLevelComponent.cs

```diff
@@ -0,0 +1,7 @@
+namespace Content.Shared._RMC14.Camera;
+
+[RegisterComponent]
```

Comment:
This should be NetworkedComponent unless you want the client to add it independently